### ISSUE TITLE
EDI 820 file

### ISSUE THREAD

**Issue by sanvaibhav** [BerryWorksSoftware/edireader#13]
Can i parse EDI 820 file using this ?

**Comment by canabrook** [comment]
Yes, certainly.  An 820 (Payment Order) may be parsed at any version (4010, etc). It may be transformed into XML using this project, or into JSON using the EDI to JSON companion project. Later in the day I will respond with a simple example, showing an 820 as XML.
